### PR TITLE
added default lat/lng fields for mapCenter if no results found

### PR DESCRIPTION
**Pull request by mgratch**

### CODE REVIEW

**Comment by mgibbs189** [comment]
Thanks Mark!

@Desertsnowman could you review? Does everything here look OK?

File: settings/settings.php [hunk 1]
```diff
@@ -80,6 +80,32 @@
                         ),
                     ),
                 ),
+                array(
+                    'column' => array(
```
Comment (Desertsnowman) [inline settings/settings.php]: @mgratch +10 points for figuring this out!

Comment (Desertsnowman) [inline settings/settings.php]: not that it was hard, but really nice to see someone use something I made.

Comment (mgratch) [inline settings/settings.php]: It was very easy to get around.  Very nicely coded!

**Comment by mgibbs189** [comment]
All systems go! Thanks @mgratch

**Pull request merged**